src/ontology/uberon-edit.obo

```diff
@@ -225642,18 +225646,6 @@ is_a: UBERON:0001744 ! lymphoid tissue
 relationship: part_of UBERON:0010417 ! lymph node T cell domain
 created_by: http://orcid.org/0000-0002-2825-0621
 
-[Typedef]
-id: RO:0002473 ! composed primarily of
-alt_id: UBREL:0000002
```

Comment:
Do we need to make sure these alt_Ids are preserved with declarations + replaced_by annotation axioms?   - see also BFO IDs below.
(CC @cmungall )